types/ace/index.d.ts

```diff
@@ -114,22 +114,22 @@ declare namespace AceAjax {
 
         /**
          * Sets a Configuration Option
-        **/
+       **/
```

Comment:
I normally ignore unrelated formatting changes in DT PRs, but all 800 or so of these comment closing lines are now misaligned 🥴